synapse/tests/test_lib_stormtypes.py

```diff
@@ -1834,6 +1834,7 @@ async def test_storm_lib_layer(self):
                 layr = core.getLayer(locklayr)
                 self.true(layr.lockmemory)
 
+            # FIXME Why was the remainder of this disabled?
```

Comment:
can't merge a FIXME right?

Reply:
nope and its entirely unclear why this test was left mangled and incomplete too